source/routing/benchmark_full.cpp

```diff
@@ -162,6 +162,7 @@ int main(int argc, char** argv) {
             ("verbose,v", "Verbose debugging output")
             ("nb_second_pass", po::value<int>(&nb_second_pass)->default_value(0), "nb second pass")
             ("stop_files", po::value<std::string>(&stop_input_file), "File with list of start and target")
+            ("dump_demands", po::value<std::string>(&demands_output_file), "Write a csv file with the list of demands (start, target, time) used")
```

Comment:
minor on vocable used in navitia for journeys:
```suggestion
            ("dump_requests", po::value<std::string>(&demands_output_file), "Write a csv file with the list of demands (from, to, datetime) used")
```

Reply:
Corrected in 78fafeb2b78701bf233a101456b4828701b9e8e4